lib/roo_on_rails/routemaster/lifecycle_events.rb

```diff
@@ -31,7 +30,7 @@ def publish_event(event, force_publish:)
           begin
             publisher.publish!(force_publish: force_publish)
           rescue => e
-            NewRelic::Agent.notice_error(e)
+            Raven.report_exception(e) if defined?(Raven)
```

Comment:
Might also be worth logging with `Rails.logger` so it's still recorded in Datadog for services without Sentry?